shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -417,7 +416,7 @@ static bool ClipRRectContainsPlatformViewBoundingRect(const SkRRect& clip_rrect,
 
 long FlutterPlatformViewsController::FindFirstResponderPlatformViewId() {
   for (auto const& [id, root_view] : root_views_) {
-    if ((UIView*)(root_view.get()).flt_hasFirstResponderInViewHierarchySubtree) {
+    if (((UIView*)root_view.get()).flt_hasFirstResponderInViewHierarchySubtree) {
```

Comment:
`error: cast of 'BOOL' (aka 'bool') to 'UIView *' is disallowed with ARC`
```
../../../flutter/shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm:419:9: error: cast of 'BOOL' (aka 'bool') to 'UIView *' is disallowed with ARC
  419 |     if ((UIView*)(root_view.get()).flt_hasFirstResponderInViewHierarchySubtree) {
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```
so... was this ever working?

Reply:
weird